plans/common.sh

```
@@ -0,0 +1,68 @@
+    echo "=== FAILED — full output ==="
+    cat "${_TMPLOG}"
+  fi
+
+  rm -f "${_TMPLOG}"
+  _TMPLOG=""
+  return ${exit_code}
+}
+
+collect_artifact() {
+  local artifact
+  artifact=$(ls envoy-alpha-*.tar.gz 2>/dev/null | head -1)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)